module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -121,13 +129,39 @@ namespace module::purpose {
             }
         });
 
+        // Left button pauses the soccer scenario
+        on<Trigger<ButtonLeftDown>, Single>().then([this] {
+            emit<Scope::DIRECT>(std::make_unique<ResetFieldLocalisation>());
+            emit<Scope::DIRECT>(std::make_unique<EnableIdle>());
+            emit<Scope::DIRECT>(std::make_unique<Buzzer>(1000));
+            idle = true;
+        });
+
+        on<Trigger<ButtonLeftUp>, Single>().then([this] { emit<Scope::DIRECT>(std::make_unique<Buzzer>(0)); });
+
+        on<Trigger<EnableIdle>, Single>().then([this] {
+            // Stop all tasks and stand still
+            emit<Task>(std::unique_ptr<FindPurpose>(nullptr));
+            emit(std::make_unique<Stability>(Stability::UNKNOWN));
+            log<NUClear::INFO>("Idle mode enabled");
```

Comment:
```suggestion
            log<NUClear::DEBUG>("Idle mode enabled");
```

Reply:
I think it's nice to see that button has been pressed and certain mode is enabled

Reply:
I think these should be INFO